CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
 * Enhance `KafkaBridge` resource with consumer inactivity timeout and HTTP consumer/producer enablement.
 * Add support for feature gates to User and Topic Operators
 
+### Changes, deprecations and removals
+
+* The `reconciliationIntervalSeconds` configuration for the Topic and User Operators is deprecated, and will be removed when upgrading schemas to V1. Use `reconciliationIntervalMs` converting the value to milliseconds.
```

Comment:
Just a nit - we would refer to it as `v1` and not `V1`. Should we also mention that we move to milliseconds because that is used in the actual configuration of the operators as an explanation?

```suggestion
* The `reconciliationIntervalSeconds` configuration for the Topic and User Operators is deprecated, and will be removed when upgrading schemas to v1. Use `reconciliationIntervalMs` converting the value to milliseconds.
```

Reply:
Updated to v1. I'm saying to convert to ms. Is it clear enough?